src/app/modules/widgets/tree-viewer/tree-viewer.component.ts

```diff
@@ -19,6 +20,21 @@ export class TreeViewerComponent implements OnInit {
   @Input() level = 0;
   @Input() config: TreeViewerConfig;
 
+  _newElementConfig: NewElementConfig;
+
+  get newElementConfig(): NewElementConfig {
+    if (!this._newElementConfig) {
+      this._newElementConfig = {
+        indent: this.isLeaf ? '0px' : '12px',
```

Comment:
should depend on context being a folder or not!
how about putting the ContextType into the configuration?

Reply:
indent is now a boolean and part of the msg bus event payload